### PR TITLE
Fix the oversight from #18841

### PR DESCRIPTION
**Pull request by lidizheng**
Related: #18841

BUILD ID: 26211204-a0c3-481e-8465-8d1542b26cdc
Waiting the grpc_interop_matrix_adhoc build to pass.

### CODE REVIEW

File: tools/interop_matrix/client_matrix.py [hunk 1]
```diff
@@ -266,8 +266,8 @@ def __init__(self, patch=[], runtime_subset=[], testcases_file=None):
         ('v1.15.0', ReleaseInfo(testcases_file='csharp__v1.3.9')),
         ('v1.16.0', ReleaseInfo(testcases_file='csharp__v1.3.9')),
         ('v1.17.1', ReleaseInfo(testcases_file='csharp__v1.3.9')),
-        ('v1.18.0', ReleaseInfo(testcases_file='csharpcoreclr__v1.18.0')),
-        ('v1.19.0', ReleaseInfo(testcases_file='csharpcoreclr__v1.18.0')),
+        ('v1.18.0', ReleaseInfo(testcases_file='csharp__v1.18.0')),
```
Comment (jtattermusch) [inline tools/interop_matrix/client_matrix.py]: this is now correct

File: tools/interop_matrix/testcases/csharp__v1.18.0 [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```
Comment (jtattermusch) [inline tools/interop_matrix/testcases/csharp__v1.18.0]: there actually need to be two files - csharp__v1.18.0  (which is a copy of the original csharp__master and csharpcoreclr__v1.18.0, which is a copy of old csharpcoreclr__master).

The current contents of the file are wrong (netcoreapp1.1 corresponds to CoreCLR called "csharpcoreclr" here, not to .NET desktop called "csharp" here)

See this snippet to understand how this works
https://github.com/grpc/grpc/blob/a3062ca04afef15327512d343cbd1768326b7500/tools/interop_matrix/run_interop_matrix_tests.py#L141

**Comment by lidizheng** [comment]
@jtattermusch PTAL. The `csharp__v1.18.0` is copied from earlier version `csharp__master`.

New BUILD ID: e08a8bf5-2688-4a83-b1f9-c827406e813c

File: tools/interop_matrix/testcases/csharp__v1.18.0 [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+# DO NOT MODIFY
+# # This file is generated by run_interop_tests.py/create_testcases.sh
+# echo "Testing ${docker_image:=grpc_interop_csharp:71b05977-476b-4e57-9752-dd211c9e3741}"
+# docker run -i --rm=true -w /var/local/git/grpc/src/csharp/Grpc.IntegrationTesting.Client/bin/Debug/net45 --net=host $docker_image bash -c "mono Grpc.IntegrationTesting.Client.exe --server_host=grpc-test.sandbox.googleapis.com --server_port=443 --test_case=large_unary --use_tls=true"
```
Comment (jtattermusch) [inline tools/interop_matrix/testcases/csharp__v1.18.0]: why is it commented out?

**Comment by lidizheng** [comment]
@jtattermusch That's awkward. PTALA.

New BUILD ID: 45c0120c-7d6b-4d1a-8312-f18bdeaf1b98

**Comment by lidizheng** [comment]
Test passed, merging.

**Pull request merged**